wrappers/IWearActuators/src/IWearActuatorsWrapper.cpp

```diff
@@ -59,6 +62,11 @@ bool IWearActuatorsWrapper::open(yarp::os::Searchable& config)
         return false;
     }
 
+    if (!config.check("gloveActuatorCommandInputPortName") || !config.find("gloveActuatorCommandInputPortName").isString()) {
+        yError() << LogPrefix << "gloveActuatorCommandInputPortName parameter not found";
+        return false;
+    }
```

Comment:
Maybe it is better to have a default instead here. In this way, old configuration files will not break.

Reply:
do you mean something like this:
```c++
    if (!config.check("gloveActuatorCommandInputPortName") || !config.find("gloveActuatorCommandInputPortName").isString()) {
        yWarning() << LogPrefix << "gloveActuatorCommandInputPortName parameter not found! Using the default one.";
        pImpl->gloveActuatorCommandInputPortName = "/WearableData/HapticGlove/RightHand/Actuators/input:i";
        return true;
    }
```

The problem here is that I need to find a way to specify the hand side part in the default port name.

Reply:
with https://github.com/robotology/wearables/pull/204/commits/12b55ac363c68d0e7445ca212bdd73a87b63d3ba I tried to specify the right and left hand's default actuator port name by finding the hand side parameter in the `IWearActuatorsWrapper.cpp`.